ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.18.yaml

```diff
@@ -783,6 +783,20 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb
+- as: vsphere-ipi-multi-vcenter-f28
+  cron: 35 13 5 * *
+  steps:
+    cluster_profile: vsphere-elastic
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-vsphere-ipi-multi-vcenter
+- as: vsphere-ipi-multi-vcenter-f28-destructive
```

Comment:
This is upgrade testing, not e2e testing, pls refer to the existing jobs to figure out how to cover the install profile.